storage/localstore/mode_get.go

```diff
@@ -143,6 +145,101 @@ func (db *DB) updateGC(item shed.Item) (err error) {
 	return db.shed.WriteBatch(batch)
 }
 
+func (db *DB) IsHashPresentInRetrievalDataIndex(hash []byte) bool {
+	var item shed.Item
+	item.Address = make([]byte, len(hash))
+	copy(item.Address[:], hash[:])
+	_, err := db.retrievalDataIndex.Get(item)
+	return !(err != nil)
+}
+
+func (db *DB) IsPinnedFileRaw(hash []byte) (bool, error) {
+	var item shed.Item
+	item.Address = make([]byte, len(hash))
+	copy(item.Address[:], hash[:])
+	i, err := db.pinFilesIndex.Get(item)
+	if err != nil {
+		return false, err
+	}
+	raw := false
+	if i.IsRaw > 0 {
+		raw = true
+	}
+	return raw, nil
+}
+
+func (db *DB) IsFilePinned(hash []byte) bool {
```

Comment:
Why not use chunk.Address instead of []byte?

Reply:
Done. Same as above